packages/backend-core/src/index.ts

```diff
@@ -21,6 +21,7 @@ export * as context from "./context"
 export * as cache from "./cache"
 export * as objectStore from "./objectStore"
 export * as redis from "./redis"
+export { Client as RedisClient } from "./redis"
```

Comment:
Probably wise to be explicit about this export anyway, but just an FYI that it can already be consumed as:
```ts
import { redis } from "@budibase/backend-core"
let socket: redis.Client
```
But I definitely prefer the explicit syntax!

Reply:
In this case I'm explicitly exporting the type, we've had issues with this in the past, but you need to export from the index of backend-core for the types to be consumable.